examples/grpc/buf.gen.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+version: v1
+plugins:
+  - plugin: go
+    out: .
+    opt:
+      - paths=source_relative
+  - plugin: go-grpc
+    out: .
+    opt:
+      - paths=source_relative
+  - plugin: python
+    out: plugin-python
+  - plugin: buf.build/grpc/python:v1.58.1
```

Comment:
So easy 😍. If you want, we could use remote plugins the same way for the go and go-grpc plugins and drop those tools from this repo. Usually the argument against it is that we don't want to depend on the BSR but since we're doing this for python anyway, why not?

Reply:
Yeah I think that would be a nice win to make it easy to use consistent plugin versions too. Added in e340950.